sale_stock_line_sequence/models/sale_order.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def _update_moves_sequence(self):
+        for order in self:
+            if any(
+                [
+                    ptype in ["product", "consu"]
+                    for ptype in order.order_line.mapped("product_id.type")
+                ]
+            ):
+                for picking in order.picking_ids:
+                    for move in picking.move_lines:
+                        if move.sale_line_id.display_type:
+                            continue
+                        move.write({"sequence": move.sale_line_id.visible_sequence})
```

Comment:
Error while confirming order

![image](https://github.com/OCA/sale-workflow/assets/94696387/d13783cc-c524-4cc7-8348-fa46ea790ce6)

Reply:
It needs the PR https://github.com/OCA/sale-workflow/pull/2636 to be merged before this can be tested.

Reply:
Now, it should be ready to review